lib/zendesk_apps_support/package.rb

```diff
@@ -131,42 +126,64 @@ def readified_js(app_name, app_id, asset_url_prefix, settings = {}, locale = 'en
           asset_url_prefix: asset_url_prefix,
           app_class_name: app_class_name,
           author: author,
-          translations: app_translations(locale),
+          translations: translations[locale],
           framework_version: framework_version,
           templates: templates,
-          settings: settings,
-          app_id: app_id,
           modules: commonjs_modules
       )
     end
 
-    def customer_css
-      css_file = file_path('app.css')
-      File.exist?(css_file) ? File.read(css_file) : ''
+    def manifest_json
+      @manifest ||= read_json('manifest.json')
     end
 
-    def has_js?
-      file_exists?('app.js')
+    def requirements_json
+      return nil unless has_requirements?
+      @requirements ||= read_json('requirements.json')
     end
 
-    def has_lib_js?
-      lib_files.any?
+    def no_template
+      if manifest_json['noTemplate'].is_a?(Array)
```

Comment:
yes, can we do this in a different PR?
There is so much more I want to fix (no_template?)